contracts/scryptVerifier.sol

```diff
@@ -8,6 +8,16 @@ import {ScryptFramework} from "./scryptFramework.sol";
   * @author Christian Reitwiessner
 */
 contract ScryptVerifier is ScryptFramework {
+  /**
+    * @dev verifies a step
+  *
+    * @param step the index of the step to verify
+    * @param preState the previous state's serialized State struct instance
+    * @param postState the current state's serialized State struct instance
```

Comment:
I think we should not use the word 'current' in this context. This function verifies the transition from the state before step `step` to the state after step `step`.